charts/core/src/core.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storybookArgTypes } from '@lg-tools/storybook-utils';
+import { StoryFn } from '@storybook/react';
+
+import { LineProps } from './Line';
+import { makeLineData } from './testUtils';
+import { Chart, Grid, Line } from '.';
+
+export default {
+  title: 'Charts/Core',
+  component: Chart,
+  argTypes: {
+    darkMode: storybookArgTypes.darkMode,
+    data: {
+      control: {
+        type: 'object',
+      },
+    },
+    verticalGridLines: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    horizontalGridLines: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+  decorator: (Instance, context) => {
+    return <Instance darkMode={context?.args.darkMode} />;
+  },
+};
+
+interface StoryChartProps {
+  data: Array<LineProps>;
+  verticalGridLines: boolean;
+  horizontalGridLines: boolean;
+}
+
+const Template: React.FC<StoryChartProps> = props => {
+  const { data, verticalGridLines, horizontalGridLines } = props;
+  return (
+    <Chart {...props}>
+      <Grid vertical={verticalGridLines} horizontal={horizontalGridLines} />
+      {data.map(({ name, data }) => (
+        <Line name={name} data={data} key={name} />
+      ))}
+    </Chart>
+  );
+};
+
+export const LiveExample: StoryFn<StoryChartProps> = Template.bind({});
+LiveExample.args = {
+  data: makeLineData(10),
+  horizontalGridLines: true,
+  verticalGridLines: false,
+};
```

Comment:
If you only have one story, you don't necessarily need to declare template and to all this